Mathlib/NumberTheory/KummerDedekind.lean

```diff
@@ -224,8 +224,6 @@ noncomputable def quotAdjoinEquivQuotMap (hx : (conductor R x).comap (algebraMap
     · exact Ideal.Quotient.mk_surjective
 #align quot_adjoin_equiv_quot_map quotAdjoinEquivQuotMap
 
--- Porting note: on-line linter fails with `failed to synthesize` instance
--- but #lint does not report any problem
```

Comment:
Let me also flag this one: this was nolint simpNF before, right? Out of context, this change looks slightly odd.